lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -120,42 +122,44 @@ def allocate_load_balancer(action_handler, lb_spec, lb_options, machine_specs)
           end
         end
 
-        # Update listeners
-        perform_listener_action = proc do |desc, &block|
```

Comment:
@jkeiser I didn't see the use in having both this and `perform_action` - what am I missing?